### PR TITLE
Update CONTRIBUTORS.md

### PR DESCRIPTION
**Pull request by mariamuradd**

### CODE REVIEW

File: CONTRIBUTORS.md [hunk 1]
```diff
@@ -1,3 +1,3 @@
 # Contributors
-
+[maria murad](https://github.com/mariamuradd)
```
Comment (blwatkins) [inline CONTRIBUTORS.md:2]: Could you please move this to line 3 or 4 and leave line 2 blank?

**Review by blwatkins — CHANGES_REQUESTED** [review]
One small change to make, but overall it looks good.

**Comment by mariamuradd** [comment]
changed to line 4!

**Comment by blwatkins** [comment]
I believe if you perform a pull request of your main branch into your patch-1 branch, I should be able to see the change here.  You can also close this pull request and open a new one with your changes, if you would prefer.

When everything has been updated correctly, you should be able to see your changes in the "Files Changed" tab of the pull request.

**Pull request closed without merging**